src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System;
-using System.Linq;
-using System.Windows.Controls;
-using Dynamo.Extensions;
+﻿using Dynamo.Extensions;
 using Dynamo.Graph.Workspaces;
 using Dynamo.Logging;
 using Dynamo.PackageManager;
 using Dynamo.WorkspaceDependency.Properties;
 using Dynamo.Wpf.Extensions;
+using System;
+using System.Linq;
+using System.Windows.Controls;
```

Comment:
https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards#usings I think we agreed from the recent conversation